src/webui/api/torrentscontroller.cpp

```diff
@@ -560,19 +560,25 @@ void TorrentsController::filePrioAction()
 
     const QString hash = params()["hash"];
     const QStringList fileIDs {params()["id"].split('|')};
-    bool ok;
+    bool ok = false;
     const int priority = params()["priority"].toInt(&ok);
     if (!ok)
-        throw APIError(APIErrorType::BadParams, tr("Priority must be an int"));
+        throw APIError(APIErrorType::BadParams, tr("Priority must be an integer"));
 
     BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(hash);
-    if (torrent && torrent->hasMetadata()) {
-        for (const auto &fileID : fileIDs) {
-            const int id = fileID.toInt(&ok);
-            if (ok)
-                torrent->setFilePriority(id, priority);
-        }
+    if (!torrent || !torrent->hasMetadata())
+        throw APIError(APIErrorType::BadParams, tr("Torrent hash is not valid"));
+
+    QList<int> fileIdsInts;
+    for (const QString &fileID : fileIDs) {
+        const int id = fileID.toInt(&ok);
+        if (!ok)
+            throw APIError(APIErrorType::BadParams, tr("File ids must be integers"));
+        fileIdsInts << id;
     }
+
+    for (const auto &id : fileIdsInts)
```

Comment:
`const int id : fileIdsInts`

Reply:
From the coding guidelines:

`Declarations for which one can gather enough information about the object interface (type) from its name or the usage pattern .. or the right part of the expression nicely fit here.`

Reply:
The variable name does not matter. This paragraph means something completely different:
```c++
auto *varName = static_cast<RSS::Feed *>(rssItemPtr);
```
Or:
```c++
auto *session = BitTorrent::Session::instance();
```